.github/workflows/push_complement_image.yml

```diff
@@ -0,0 +1,36 @@
+# This task does not run complement tests, see tests.yaml instead.
+# This task does not build docker images for synapse for use on docker hub, see docker.yaml instead
+
+name: Store complement synapse image in ghcr.io
+on:
+  push:
+    branches: ["master", "develop"]
```

Comment:
Can you add per-branch concurrency limits here, do limit GHA worker usage when merge a bunch of PRs in short order? See e.g. https://github.com/matrix-org/sytest/blob/bebf431edf7311d5147139f44dab97131a784d29/.github/workflows/pipeline.yml#L7-L12 from sytest